src/lib/eina/meson.build

```diff
@@ -302,10 +307,18 @@ if cc.has_header_symbol(pthread_np_header_file, pthread_setname_np_api, args : p
    eina_config.set('EINA_HAVE_PTHREAD_SETNAME', '1')
 endif
 
+if cc.has_header_symbol('precessthreadspi.h', 'SetThreadDescription')
+   eina_config.set('EINA_HAVE_WIN32_THREAD_SETNAME', '1')
+endif
+
 if cc.has_header_symbol(pthread_np_header_file, 'pthread_attr_setaffinity_np', args : pthread_args)
    eina_config.set('EINA_HAVE_PTHREAD_AFFINITY', '1')
 endif
 
+if cc.has_header_symbol('WinBase.h', 'SetThreadAffinityMask')
+   eina_config.set('EINA_HAVE_WIN32_THREAD_AFFINITY', '1')
+endif
+
```

Comment:
Maybe we should add a `if host_machine.system() == 'windows'` check, what do you think?

Reply:
We did like this because we fought it was eina pattern. Does it even exist such pattern?

Reply:
The problem is the header may not exist. Lemme see how does meson deal with that.

Reply:
Ok, checked. Meson just issues this when (re)configuring:

```text
Header <nonexistent.h> has symbol "some_symbol" : NO
```